.golangci.yml

```diff
@@ -4,7 +4,7 @@ run:
   concurrency: 4
 
   # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 20m
+  timeout: 100m
```

Comment:
is this still necessary?

Reply:
My apologies, all of the changes you marked were a part of my testing to determine the failure point in the linting process. I'm going to revert as many of these changes as possible, and mark the review as `Ready for Review` when it's ready again.

Reply:
No apologies necessary :) happy to review when it's good to go